simul/runfiles/nottest_byzcoin_ntree.toml

```diff
@@ -0,0 +1,7 @@
+Servers = 5
+Simulation = "ByzCoinNtree"
+Rounds = 2
+BF = 2
```

Comment:
A general remark as this seem to happen quite often: It's strange that files you rename appear as deleted (in one commit) and create in another. It's difficult to see the changes/diff if we do it like this (which is bad for reviewing the code).